.github/workflows/macos.yml

```diff
@@ -0,0 +1,27 @@
+name: macos
+
+on:
+  push:
+    branches:
+      - "master"
+    paths-ignore:
+      - "**.md"
+  pull_request:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - name: Install dependencies
+        run: brew install automake sdl2 sdl2_image sdl2_mixer sdl2_net
```

Comment:
We don't need `sdl2_image`, only `libpng` for screenshots.

Reply:
Good catch! Was a bit on autopilot there.